src/dlgProfilePreferences.cpp

```diff
@@ -128,7 +133,12 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
         checkbox_noAutomaticUpdates->setDisabled(true);
         checkbox_noAutomaticUpdates->setToolTip(tr("Automatic updates are disabled in development builds to prevent an update from overwriting your Mudlet"));
     } else {
-        checkbox_noAutomaticUpdates->setChecked(!mudlet::self()->updater->updateAutomatically());
+        checkbox_noAutomaticUpdates->setChecked(!pMudlet->updater->updateAutomatically());
+        // This is also one of the group of signal/slot connections that handle
+        // updating one profile's preferences form/dialog when a different
+        // profile saves new settings that are application wide - so that this
+        // instance takes on the new settings for those controls
```

Comment:
This comment needs to be simplified as well - isn't it just saying "automatically update all other open preferences windows"? Why does it need to be a comment to begin with - it should be obvious what it does from the code. Comments are for things where the code can't be made intuitive.

Reply:
I had to leave it away from the other added ones because it is only needed when the updater code is compiled in - but yeah - the comment here and the one above are overlong... :slightly_frowning_face: